momentumopt/python/momentumopt/motion_planner.py

```diff
@@ -277,11 +278,15 @@ def time_vector(self):
     def optimize_motion(self, plot_com_motion=True):
         dyn_optimizer = self.dyn_optimizer
         kin_optimizer = self.kin_optimizer
-
+        #dyn_optimizer.dynamicsSequence().dynamics_states[0].setCoM2(100, 100, 100)
+        dyn_optimizer.dynamicsSequence().setCoMAtTime(np.ones(3), 0)
+        print dyn_optimizer.dynamicsSequence().dynamics_states[0].com
```

Comment:
please remove the print.
BTW are you using python2 or 3?
because this is python2 code...
Please update to python3 I can help you if needed.